test/source/tests/gmail.ts

```diff
@@ -44,29 +43,35 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
     };
 
     const createSecureDraft = async (t: AvaContext, browser: BrowserHandle, gmailPage: ControllablePage, content: string, params: { offline: boolean } = { offline: false }) => {
-      const urls = await gmailPage.getFramesUrls(['/chrome/elements/compose.htm'], { sleep: 1 });
-      const composeBox = await browser.newPage(t, urls[0]);
+      let composeBox;
```

Comment:
can we define the `composeBox` of type `Controllable | undefined` (or `ControllableBase | undefined`)
to prevent the possibility of other types accidentally mixing in?

Reply:
Done, thanks 👍